OtripleS.Web.Api.Tests.Unit/Services/Foundations/Classrooms/ClassroomServiceTests.cs

```diff
@@ -78,6 +78,19 @@ public static IEnumerable<object[]> InvalidMinuteCases()
             };
         }
 
+        public static T GetInvalidEnum<T>()
+        {
+            int randomNumber = GetLocalRandomNumber();
+            while(Enum.IsDefined(typeof(T), randomNumber) is true)
```

Comment:
Need a line break between single and multi-line

Reply:
It's fixed.